build-system/server.js

```diff
@@ -456,6 +466,13 @@ app.use('/min/', function(req, res) {
   proxyToAmpProxy(req, res, /* minify */ true);
 });
 
+// Nest the response in an iframe.
+// Example:
+// http://localhost:8000/iframe/examples/ads.amp.max.html
+app.use('/iframe/', function(req, res) {
```

Comment:
i guess you need to strip out the leading '/iframe/'? you can do that by
also, use `app.get` instead of `app.use`